bcipy/task/paradigm/rsvp/calibration/calibration.py

```diff
@@ -93,6 +95,36 @@ def generate_stimuli(self):
                                              is_txt=self.rsvp.is_txt_stim,
                                              color=self.color)
 
+    def convert_timing_triggers(self, timing: List[tuple]) -> List[Trigger]:
```

Comment:
This method could be a function, which would make it easier to add some tests for it.

Reply:
This seems like something all rsvp tasks may need and with slightly different trigger typing logic... I can add tests to it here too! How were you thinking of pulling it out?

Reply:
I was thinking it could be a function in this same file since it doesn't need anything specific from the class (property or other method call) for doing the conversion.

In looking at the difference between this function and the one used by CopyPhrase, the unique thing is how we determine a trigger type (given a symbol, target, and position), so maybe that's the task specific code.